src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -116,7 +118,7 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      *
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
-    public AddressBook load() throws StorageOperationException {
+    public AddressBook load() throws StorageOperationException, FileNotFoundException {
```

Comment:
Purpose is to check accidental file deletes and not check missing file initially so load() does not make such sense.